woodwork/tests/accessor/test_statistics.py

```diff
@@ -291,7 +267,7 @@ def test_describe_accessor_method(describe_df):
         assert isinstance(stats_df, pd.DataFrame)
         assert set(stats_df.columns) == {'category_col'}
         assert stats_df.index.tolist() == expected_index
-        pd.testing.assert_series_equal(expected_vals, stats_df['category_col'].dropna())
+        expected_vals.equals(stats_df['category_col'].dropna())
```

Comment:
I don't understand why, but the previous assertion using `pd.testing.assert_series_equal` fails for the Koalas test with an error `TypeError: 'set' object is not subscriptable`. I wasn't able to identify any differences between the stats_df we were getting with Koalas and the one from pandas/Dask, so I'm not sure what is causing the issue.

Had to change this here and below in order for this test to pass with Koalas.

Reply:
I feel like any time something changes in a way where we have to touch that describe test, it breaks in this way, and I can never remember what it means either. But if this change works I'm happy to keep it. 

Might be worth refactoring this test to be a bit more manageable at some point.